certified-connectors/Oneflow/script.csx

```diff
@@ -0,0 +1,778 @@
+                public bool signatory { get; set; }
+
+                [System.Runtime.Serialization.OnDeserialized]  
+                internal void OnDeserializedMethod(System.Runtime.Serialization.StreamingContext context)
+                {
+                  if (_permissions == null && updatePermission.HasValue)
+                  {
+                    _permissions = updatePermission.Value;
+                  }
+                }
+
+                [System.Runtime.Serialization.OnSerializing]
+                internal void OnSerializingMethod(System.Runtime.Serialization.StreamingContext context)
+                {
+                    if (updatePermission.HasValue) updatePermission = null;
+                }
+
+                public override bool Equals(object obj)
+                {
+                    return obj is Participant participant &&
+                           EqualityComparer<ParticipantPermissions>.Default.Equals(_permissions, participant._permissions) &&
+                           delivery_channel == participant.delivery_channel &&
+                           email == participant.email &&
+                           identification_number == participant.identification_number &&
+                           name == participant.name &&
+                           phone_number == participant.phone_number &&
+                           sign_method == participant.sign_method &&
+                           title == participant.title &&
+                           two_step_authentication_method == participant.two_step_authentication_method &&
+                           signatory == participant.signatory;
+                }
+
+                public override int GetHashCode()
+                {
+                    int hashCode = -1207568244;
+                    hashCode = hashCode * -1521134295 + EqualityComparer<ParticipantPermissions>.Default.GetHashCode(_permissions);
+                    hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(delivery_channel);
+                    hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(email);
+                    hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(identification_number);
+                    hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(name);
+                    hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(phone_number);
+                    hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(sign_method);
+                    hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(title);
+                    hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(two_step_authentication_method);
+                    hashCode = hashCode * -1521134295 + signatory.GetHashCode();
```

Comment:
@DmitryKleshchov Can you please let us know why the hashcode is defined as a static string?

Reply:
@sriyen-msft  not sure if you mistyped string with int. 

- this is initial hash, a prime number
- this is default VS code & Visual Studio implementation of override Equals() & GetHashcode() methods
- Though, it was not correct to have different set of fields for GetHashCode() and Equals, so I removed some for Party object.